UIKitTeamPlate/UIKitTeamPlate/CoffeeMenuViewController.swift

```diff
@@ -1,14 +1,309 @@
+        label.textAlignment = .right
+        label.font = UIFont(name: Constants.verdanaBoldFont, size: 18)
+        return label
+    }()
+
+    /// Кнопка формирования заказа
+    private lazy var orderButton: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 15, y: 717, width: 345, height: 53)
+        button.setTitle(Constants.orderButtonTitleLabelText, for: .normal)
+        button.titleLabel?.font = UIFont(name: Constants.verdanaBoldFont, size: 18)
+        button.layer.cornerRadius = 12
+        button.backgroundColor = .seaGreen
+        button.addTarget(self, action: #selector(makeOrder), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+
+    /// Установка вью на супервью
+    private func setupUI() {
+        navigationItem.leftBarButtonItem = createLeftBarButtonItem()
+        navigationItem.rightBarButtonItem = createRightBarButtonItem()
+
+        view.backgroundColor = .white
+        [
+            backgroundView,
+            segmentedControl,
+            modificationLabel,
+            chooseRoastButton,
+            coffeeImageView,
+            additionsButton,
+            priceLabel,
+            orderButton
+        ]
+        .forEach { view.addSubview($0) }
+
+        switch coffeeItem.roast {
+        case .dark:
+            chooseRoastButton.configuration?.image = UIImage(named: Constants.darkRoastImage)
+            chooseRoastButton.setTitle(coffeeItem.roast.rawValue, for: .normal)
+        case .light:
+            chooseRoastButton.configuration?.image = UIImage(named: Constants.lightRoastImage)
+            chooseRoastButton.setTitle(coffeeItem.roast.rawValue, for: .normal)
+        }
+
+        priceLabelText = "Цѣна - \(coffeeItem.fullPrice) руб"
+        priceLabel.text = priceLabelText
+
+        switch coffeeItem.fullPrice {
+        case coffeeItem.standardPrice:
+            additionsButton.configuration?.image = UIImage(named: Constants.addItemImage)
+        default:
+            additionsButton.configuration?.image = UIImage(named: Constants.addedItemImage)
+        }
+    }
+
+    /// Метод сброса опций модели
+    private func reloadOptions() {
+        coffeeItem.roast = .dark
+        coffeeItem.isStandardMilkAdded.0 = false
+        coffeeItem.isSyrupAdded.0 = false
+        coffeeItem.isSoyaMilkAdded.0 = false
+        coffeeItem.isAlmondMilkAdded.0 = false
+        coffeeItem.isEspressoChosen.0 = false
+    }
+
+    /// Метод возврата на один экран назад
+    @objc private func popToPrevious() {
+        navigationController?.popViewController(animated: true)
+    }
+
+    /// Метод изменения вью и модели от изменения состояния свитчера
+    @objc private func changeCoffeeType() {
+        switch segmentedControl.selectedSegmentIndex {
+        case 0:
+            coffeeItem.coffeeType = .americano
```

Comment:
```suggestion
            coffeeItem.coffeeType = .americano
```
тут лучше сделать 2 массива с кексами и названиями и доставать в 1 строку array[segmentedControl.selectedSegmentIndex]